src/components/ec-radio-btn-group/ec-radio-btn-group.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    class="ec-radio-btn-group"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-radio-btn-group` : `ec-radio-btn-group`"
+  >
+    <div
+      v-if="hasLabel"
+      data-test="ec-radio-btn-group__label"
+      class="ec-radio-btn-group__label"
+    >
+      <slot name="label">{{ label }}</slot>
+    </div>
+    <div
+      data-test="ec-radio-btn-group__radio-btn-wrapper"
+      class="ec-radio-btn-group__radio-btn-wrapper"
+      :class="{'ec-radio-btn-group__radio-btn-wrapper--is-single-line': isGroupInline}"
+    >
+      <ec-radio-btn
+        v-for="(option, optionIndex) in options"
+        :key="optionIndex"
+        :is-text-inline="isTextInline"
+        :is-disabled="isDisabled"
+        :has-error="isInvalid"
+        :model-value="modelValue"
+        :name="name"
+        :value="option.value"
+        :label="option.label"
+        :description="option.description"
+        :class="[
+          {
+            'tw-mt-16': optionIndex !== 0 && !isGroupInline,
+            'tw-ml-40': optionIndex !== 0 && isGroupInline,
+          }
+        ]"
+        @update:model-value="emit('update:modelValue', $event);"
+      />
+    </div>
+    <div
+      :id="errorId"
+      v-if="hasErrorMessage"
+      class="ec-radio-btn-group__error-text"
+      data-test="ec-radio-btn-group__error-text"
+    >
+      <slot name="error-message">{{ errorMessage }}</slot>
+    </div>
+  </div>
+
+</template>
+
+<script setup lang="ts">
+import {
+  computed,
+  useSlots,
+} from 'vue';
+
+import { getUid } from '../../utils/uid';
+import EcRadioBtn from '../ec-radio-btn';
+import type { RadioButtonGroupEvent, RadioButtonGroupEvents, RadioButtonOption } from './types';
+
+interface RadioButtonGroupProps {
+  options: RadioButtonOption[],
+  name?: string,
```

Comment:
Do we need this name? We could generate unique id and use it. Otherwise this prop should be required as we must set name for radio button group.

Reply:
it can be optional because the group can be the only one in the view

Reply:
I think we could remove this property, and generate a unique name in <script> even though it could be the only one RadioButtonGroup on the page.